frontend/src/components/Forms/variances/AddVarianceForm.js

```diff
@@ -50,6 +51,7 @@ export class AddVarianceForm extends Component {
   };
 
   render() {
+    const filesUploaded = this.state.uploadedFiles.length >= 1;
```

Comment:
Logically, this will be exactly the same, but I'm so used to seeing `> 0` that this looks weird 😆